production/helm/loki/templates/chunks-cache/poddisruptionbudget-chunks-cache.yaml

```diff
@@ -12,5 +13,6 @@ spec:
     matchLabels:
       {{- include "loki.selectorLabels" . | nindent 6 }}
       app.kubernetes.io/component: memcached-chunks-cache
-  maxUnavailable: 1
+  maxUnavailable: {{ .Values.chunksCache.maxUnavailable }}
```

Comment:
```suggestion
  {{- with .Values.chunksCache.maxUnavailable }}
  maxUnavailable: {{ .}}
  {{- end }}
```

Reply:
Is there any advantage in using that syntax? I explicitly did not copy style from the other yamls since I did not see any value in using a with loop there.

Reply:
users can set maxUnavailable: 0 in values.yaml to omit this settings in PDB. Thats why I prefer this.